src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Гаджеты</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Планшеты</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Фото</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+        <p class="card__code">Код товара: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Отзывов: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price__word">Цена:</p>
+          <p class="card__price__value">69 999 грн</p>
+        </div>
```

Comment:
1. `word` doesn't have any sence -> use title or label
2. due to bem methodology you should use `block__element--modifier`, here you have `block__element__element` 
change it to `card__price-label`, so your element would be `price-label`
3. you can create new block here
```suggestion
        <div class="card__price price">
          <p class="price__title">Цена:</p>
          <p class="price__value">69 999 грн</p>
        </div>
```
**same remark for all usages in file**